src/data/attribute.rs

```diff
@@ -129,7 +130,42 @@ pub fn store_metadata(metastruct: &Struct) {
     let metadata = process_metadata(metastruct, String::new());
     for (key, value) in metadata {
         let attr = format!("{KUADRANT_NAMESPACE}\\.{key}");
+        // stored into host_property: wasm.kuadrant.{key}
+        // example: wasm.kuadrant.identity.anonymous is how it's stored!
+        // but users would write the predicate: !auth.identity.anonymous
+        // two problems:
+        // - 1/ auth.identity doesn't resolve
+        // - 2/ the value is the string "true"
```

Comment:
>  the value is the string "true"

What is the problem  there? `auth.identity.anonymous` returns a string. We should document that and the user should deal with that. 

Besides, this something we can change from authorino, can't we? I mean, `auth.identity.anonymous` could be `bool` not string.